contracts/SportMarkets/SportsAMMUtils.sol

```diff
@@ -195,7 +195,12 @@ contract SportsAMMUtils {
         if (ISportPositionalMarket(_market).optionsCount() > uint(_position)) {
             uint[] memory odds = new uint[](ISportPositionalMarket(_market).optionsCount());
             odds = ITherundownConsumer(theRundownConsumer).getNormalizedOddsForMarket(_market);
-            oddsToReturn = odds[uint(_position)];
+            uint firstTag = ISportPositionalMarket(_market).tags(0);
+            if (sportsAMM.isMarketForSportOnePositional(firstTag) && uint(_position) > 0) {
+                oddsToReturn = 0;
```

Comment:
its 0 by default, its better code style to just set it to non zero if needed.

Reply:
if (!sportsAMM.isMarketForSportOnePositional(firstTag) || uint(_position) == 0){
 oddsToReturn = odds[uint(_position)];
}

Reply:
ah ok cool